t3f/ops.py

```diff
@@ -1132,3 +1147,51 @@ def gather_nd(tt, indices):
     tt_elements = tf.matmul(tt_elements, core_slices)
   tt_elements = tf.reshape(tt_elements, tf.shape(indices)[:-1])
   return tt_elements
+
+def renormalize_tt_cores(tt, epsilon=1e-8):
```

Comment:
(pep8) Please, use two blank lines before high-level function definition (note that it does NOT apply to class methods).